### PR TITLE
document another important waffle fact

### LINKED ISSUE #7: jyn is cute
@teloxidebot label +documentation

### PR DESCRIPTION
**Pull request by jyn514**
fixes #7

r? WaffleLapkin

### CODE REVIEW

**Comment by jyn514** [comment]
yay it's fixed!

**Review by Noratrieb — APPROVED** [review]

File: readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+jyn and waffle are both very cute!!
```
Comment (WaffleLapkin) [inline readme.md:1]: this is factually incorrect, since waffle is not cute

Comment (jyn514) [reply readme.md:1]: excuse me you are not allowed to lie on my PRs >:(

Comment (WaffleLapkin) [reply readme.md:1]: well I'm not lying!!

Comment (jyn514) [reply readme.md:1]: ok fine *headpats you so you will be cuter*

**Pull request merged**